middleware/encryptcookie/config.go

```diff
@@ -0,0 +1,75 @@
+package encryptcookie
+
+import "github.com/gofiber/fiber/v2"
+
+// Config defines the config for middleware.
+type Config struct {
+	// Next defines a function to skip this middleware when returned true.
+	//
+	// Optional. Default: nil
+	Next func(c *fiber.Ctx) bool
+
+	// Array of cookie keys that should not be encrypted
+	//
+	// Optional. Default: []
+	Except []string
+
+	// Base64 unique key to encode & decode cookies
+	//
+	// Optional. Default: Generating new key on every run
+	Key string
```

Comment:
We should add a doc block and maybe a runtime check regarding how this value needs to be at least 32 chars. Otherwise the code will panic on cookie creation

Reply:
I'm not sure about runtime checks because of `aes.NewCipher` checks for key lengths.
Also, it's possible to use a key with 16/24 characters to use `AES-128`/`AES-192`.